samples/BasicXrApp/OpenXrProgram.cpp

```diff
@@ -510,13 +512,16 @@ namespace {
                 createInfo.pose = poseInScene;
                 createInfo.time = placementTime;
 
-                XrResult r = xrCreateSpatialAnchorMSFT(m_session.Get(), &createInfo, hologram.Anchor.Put());
+                auto xrCreateSpatialAnchorMSFT = GET_XR_PROC(m_instance.Get(), xrCreateSpatialAnchorMSFT);
```

Comment:
When an extension function is only called once, I think it makes sense to look up the function where it is needed, but if it is called multiple times, I think it is better to cache it in a member variable at startup.